src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -449,40 +577,63 @@ void CoilCoolingDX::getDataAfterSizing(Real64 &_normalModeRatedEvapAirFlowRate,
     _normalModeRatedCapacity = this->performance.normalMode.ratedGrossTotalCap;
 }
 
+CoilCoolingDXCurveFitSpeed & CoilCoolingDX::normModeNomSpeed()
+{
+    return this->performance.normalMode.speeds[this->performance.normalMode.nominalSpeedIndex];
+}
+
+CoilCoolingDXCurveFitSpeed & CoilCoolingDX::altModeNomSpeed()
+{
+    return this->performance.alternateMode.speeds[this->performance.alternateMode.nominalSpeedIndex];
+}
+
+Real64 CoilCoolingDX::condMassFlowRate(bool const useAlternateMode)
+{
+    if (useAlternateMode) {
+        return this->altModeNomSpeed().condenser_air_flow_rate;
+    } else {
+        return this->normModeNomSpeed().condenser_air_flow_rate;
+    }
+}
```

Comment:
This is volume flow rate, at least I didn't see it converted to mass, and the function name is condMassFlowRate.